### DISCUSSION TITLE
Just a showcase

### DISCUSSION THREAD

**Discussion by vdegenne** [WayfireWM/wf-shell#440] [Show and tell]
Sorry not an issue, just sharing something I've been able to achieve thanks to wf-shell and showing its potential. I'm also very thankful to this community and I want to share.

During my sessions learning japanese I have a local server I can send vocabulary I want to save to. This server also has a route that can be used to request the last 20 words. With a little bit of html and pango tweaking I'm able to poll request this words cheet at the bottom of my screen for daily rehearsal.

![image](https://github.com/WayfireWM/wf-shell/assets/2827383/7222a3b5-493e-4433-a2ff-f655f2f001bc)

↑ it updates live every time I add a new word.

**Comment by soreau** [comment]
This fits better in the discussion section, so moving it there.
